service-api/src/main/java/greencity/dto/location/LocationSummaryDto.java

```diff
@@ -0,0 +1,19 @@
+package greencity.dto.location;
+
+import lombok.*;
+
+import java.util.List;
+
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode
+@ToString
+@Builder
+public class LocationSummaryDto {
+    private Long regionId;
+    private String nameEn;
+    private String nameUa;
+    private List<LocationToCityDto> citiesUa;
+    private List<LocationToCityDto> citiesEn;
```

Comment:
Why you create new DTO when similar is already exist ? If you will need to add field to some DTO, just extend from the existing DTO that contains similar fields and add needed field.

Reply:
There are only one fittable dto present -> LocationDto which contains a single regionId field, no other dtos extend the LoationsDto for regionId. Did in the same way.